azure-pipelines.yml

```diff
@@ -13,6 +13,35 @@ steps:
 - script: pip install -U iotedgedev
   displayName: 'Install IoT Edge Dev Tool'
 
+- task: DotNetCoreCLI@2
```

Comment:
Nit: If you remove the build step, it will still work as the `dotnet test` command builds anyway.

Reply:
Done, thank you!